templates/building.html

```diff
@@ -0,0 +1,67 @@
+{#-#################################################################
+<File Name>
+  building.html
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Apr 06, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Template for building page. Uses option grid and option form container macros
+  and passes a local building form (defined in local form_content macro).
+
+#################################################################-#}
+{% import '_macros.html' as macros %}
+{% extends "base.html" %}
+
+{#- BEGIN: Building form macro -#}
+{% macro form_content(data=None, show_add_btn=False, show_rm_btn=True) %}
+  <div class="form-group">
+    <label>I build my software using this command</label>
+    <input type="text" class="form-control" value="{{ data.get('cmd') if data }}" name="build_cmd[]">
+  </div>
+  <div class="form-group">
+    <button type="button" class="copy-btn btn btn-outline-warning {{'d-none' if not show_add_btn}}">Add and continue on this page</button>
+    <button type="button" class="copy-btn btn btn-outline-primary {{'d-none' if not show_add_btn}}" data-submit="building-form">Add and continue on next page</button>    <button type="button" data-toremove=".opt-form" class="rm-btn btn btn-outline-danger {{'d-none' if not show_rm_btn}}">Remove</button>
+  </div>
+{% endmacro %}
+{#- BEGIN: Building form macro -#}
+
+{#- BEGIN: Rendered content -#}
+{% block content %}
+  {{ macros.progress_bar(value=30) }}
+  <hr>
+  <h1>How do you build your software?</h1>
```

Comment:
Don't mean to be pedantic, but `python setup.py sdist/bdist` is **packaging**. `setup.py build` is closer to building, but there's still a lot of debate of if its *really* building. There is a whole rabbit hole you can go down if you read on it in the case of reproducible builds.

Reply:
It can probably be classified as packaging, but it's slightly complicated because the standard Python packaging tools combine the build and packaging steps.  A bunch of files are added to a build/ directory when you run python setup.py bdist.  The packaging user guides/documentation even use the word "building"
https://setuptools.readthedocs.io/en/latest/setuptools.html#developer-s-guide

And I think the Python community is working to separate the build and distribution steps judging from discussions on the Distutils forum.  Maybe in-toto doesn't need to worry about the distinction, or just define "build" (if it hasn't already).

Reply:
Will be fixed with https://github.com/in-toto/layout-web-tool/issues/2